README.md

```diff
@@ -17,9 +18,12 @@ You can view a presentation on how Pact can work in a mobile context here: [Yow!
 Implements [Pact Specification v2][pact-spec-v2],
 including [flexible matching][pact-flexible-matching].
 
-This DSL relies on the Ruby [pact-ruby-standalone][pact-mock-service-standalone] to provide the mock service for the tests.
+This DSL relies on the Ruby [pact-ruby-standalone][pact-mock-service-standalone] ([brew tap][pact-ruby-standalone-homebrew]) to provide the mock service for the tests.
+
+ℹ A framework implementing [Pact Specification v3][pact-spec-v3] is in active development and can be found at [PactSwift][pact-swift].
```

Comment:
I was thinking this could be the first line in the README to make it really prominent. Not sure if this is over the top?

**ℹ A new version featuring [Pact Specification v3][pact-spec-v3], a simplified installation and better management of the mock server processes is in active development and can be found at [PactSwift][pact-swift]. We are currently looking for people to try it out and provide feedback.**

Reply:
I thought it was over the top, that's why I've put it just under v2 link 🤷‍♂️ 
I'm not fussed. I'll move it to the top.